velero/backup-all-volumes/artifacthub-pkg.yml

```diff
@@ -0,0 +1,37 @@
+# This file contains the metadata for the policy package
+# that will be displayed on Artifact Hub
+# Some settings like the verified publisher flag or the ignored packages won't
+# be applied until the next time the repository is processed. Please keep in
+# mind that the repository won't be processed if it has not changed since the
+# last time it was processed. Depending on the repository kind, this is checked
+# in a different way. For Helm http based repositories, we consider it has
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+---
+name: backup-all-volumes
+version: 1.0.0
+displayName: Backup All Volumes
+createdAt: 2023-03-29T00:00:00.000Z
+description: |
```

Comment:
The value of the description field should be the contents of the `policies.kyverno.io/description` annotation.